javascript/javascript2/week2/main.js

```diff
@@ -0,0 +1,46 @@
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+
+const ul = document.querySelector('ul');
+
+function renderProducts(products) {
+    products.forEach(product => {
+        const li = document.createElement("li");
+        li.innerHTML = `<h3>${product.name}</h3>
+        <p>Price: ${product.price}</p>
+        <p>Rating: ${product.rating}</p>`;
+        ul.appendChild(li);
```

Comment:
I see you used a mix of 'template string' + 'element' created in legacy JS.
you can combine 
const li = `<li>
.......
</li>`
this is just a suggestion, this code is also fine for merging PR.